core/domain/role_services.py

```diff
@@ -60,6 +60,7 @@
 ACTION_EDIT_ANY_SKILL = 'EDIT_ANY_SKILL'
 ACTION_EDIT_ANY_SUBTOPIC_PAGE = 'EDIT_ANY_SUBTOPIC_PAGE'
 ACTION_EDIT_ANY_TOPIC = 'EDIT_ANY_TOPIC'
+ACTION_EDIT_ANY_BLOG_POST = 'EDIT_ANY_BLOG_POST'
```

Comment:
Do we need to use this action in any role?

Reply:
Yeah, in next PR two roles namely : blog admin and blog post editors will be done. blog admin and admin will have the right to edit any blog post.